lamindb/_record.py

```diff
@@ -263,6 +279,48 @@ def _format_django_validation_error(record: Record, e: DjangoValidationError):
         return message
 
 
+def _get_model_params(record_class) -> list[tuple[str, str]]:
```

Comment:
Can you please rename this to `_get_record_kwargs()` or `_get_record_keyword_arguments()` or `_get_record_constructor_arguments()` or something like this?

We want to clearly separate the Django Model layer also in naming from what we add on top with `Record` / `BasicRecord`.

And `params` has a specific meaning in lamindb (run params, model params as in ML model params), that's one reason for why we have Google style and not NumpyDoc style.

Reply:
Agree, done.